scripts/win_chef_zero_server.ps1

```diff
@@ -0,0 +1,27 @@
+param($CZHost, $CZPort, [Switch]$UseSSL)
+$arguments = @(
+ 'C:/opscode/chef/embedded/bin/chef-zero'
+ "-p $CZPort"
+ "-H $CZHost"
+)
+
+if ($UseSSL) {
+  $arguments += '--ssl'
+}
+
+$chef_zero_server = @{
+  FilePath = 'C:/opscode/chef/embedded/bin/ruby.exe'
+  ArgumentList = $arguments
+  PassThru = $true
+}
+
+$proc = Start-Process @chef_zero_server
+Start-Sleep 2
+
+# Steal the process ID and then inject it into the taste-tester state file
+# so it can be referenced in the plugin.
+$state = Get-Content ~/.chef/taste-tester-ref.json | ConvertFrom-JSON
+$state | Add-Member -Type NoteProperty -Name pid -Value $proc.id -Force
+$state |
```

Comment:
There's a race-condition here... to be clear. but it's probably ... OK. We should at least add a comment to that effect though.